website/docs/developer/api/api-gateway.md

```diff
@@ -1,25 +1,32 @@
 ---
-id: API gateway paths, inputs and outputs
-title: API Gateway paths, inputs and outputs
-description: Additional notes on API gateway paths, inputs and expected outputs.
+id: orca-api
+title: ORCA API Reference
+description: ORCA API reference for developers that provides API documentation and interactions.
 ---
 
 ## Overview
 
-This documentation provides details on the paths, input and outputs for the `orca_api` API gateway. The format for the API invoke URL is `https://example.execute-api.us-west-2.amazonaws.com/[stage_name]/[path]`. Check [API gateway research webpage](https://github.com/nasa/cumulus-orca/blob/master/website/docs/developer/research/research-APIGateway.md) for more information.
+The purpose of this page is to give developers information on how to use the ORCA API and explain the expected inputs, outputs and paths. The API can be used to get metadata information about a granule or a recovery job and accepts and responds with JSON payloads at various HTTPS endpoints. All ORCA APIs use the `POST` method.
 
-## Path 
-- The path `catalog/reconcile` of `orca_api` API gateway returns additional metadata information needed for reporting by triggering the `orca_catalog_reporting` lambda.
-- The path `recovery/granules` of `orca_api` API gateway returns detailed status of the granule by triggering the `request_status_for_granule` lambda.
-- The path `recovery/jobs` of `orca_api` API gateway returns detailed status for a particular recovery job by triggering the `request_status_for_job` lambda.
 
-### Inputs and outputs
-The API invoke URL can be found under `Stages` section in API gateway UI. All APIs use the `POST` method.
+## Catalog reporting API
 
-- Catalog reporting API input invoke URL example:
+The `catalog/reconcile` API call provides a user with the current listing of the ORCA catalog that can be used to reconcile granule and file information against a master catalog. For example, comparing the Cumulus primary archive holdings against the ORCA holdings to find discrepancies.
+Catalog reporting API input invoke URL example:
 `https://example.execute-api.us-west-2.amazonaws.com/orca/catalog/reconcile`
 
-- Catalog reporting API output example:
+#### Required input fields
+- pageIndex(int)- The 0-based index of the results page to return.
+- endTimestamp(int)- Cumulus granule createdAt end-time for date range to compare data, in milliseconds since 1 January 1970 UTC.
+
+#### Optional input fields
```

Comment:
Combine required and optional areas into a table instead of bullets that tells you the field name, data type , description, and if it is required or not.

Reply:
changed to a table